New log file opened at Thu Mar 16 19:40:58 PDT 2017 by 8896@DESKTOP-JI6ETD8
Taking next connection to: 59089
New client attempting to connect on port: 59090-59089
  Address = /192.168.86.106:59089
This client on port 59090 has uuid 33800431-d49d-4a45-a5c3-970556d85000 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-33800431-d49d-4a45-a5c3-970556d85000
Taking next connection to: 59089
New client attempting to connect on port: 59091-59089
  Address = /192.168.86.106:59089
Handshake exception on socket: 59091
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:171)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59089
New client attempting to connect on port: 59092-59089
  Address = /192.168.86.106:59089
This client on port 59092 has uuid b9ff9b48-60e8-4d9d-b716-9b1b1399fd57 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-33800431-d49d-4a45-a5c3-970556d85000, play-fork-run-b9ff9b48-60e8-4d9d-b716-9b1b1399fd57
Taking next connection to: 59089
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59089
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59089
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59089
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59089
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59089
Checking to see if clients are empty...
No non-daemon clients connected after 3 min. Shutting down.
1 daemon clients are connected.
Server socket thread exiting.
1 clients open, will close them...
All client sockets have been closed.
Closing the 8896@DESKTOP-JI6ETD8 logs at Thu Mar 16 20:00:06 PDT 2017, goodbye.
